docs/Agent/user-guide/man_page.md

```diff
@@ -333,107 +265,205 @@ pbench-register-tool-trigger, Registers the tool trigger
+The tool group to use for the list of tools
 
---token STR  
-pbench server authentication token [required]
+`--iteration-list`  
+A file containing a list of iterations to run for the provided script;
+the file should contain one iteration per line . With a leading '#' (hash) character used for comments and blank lines are ignored.
+Each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script;  
+_NOTE: --iteration-list is not compatible with --use-tool-triggers_
 
---delete / --no-delete  
-Remove local data after successful copy [default: delete]
+`--sysinfo` STR,[STR]  
+comma-separated values of system information to be collected; available: default, none, all, ara, block, insights, kernel_config, libvirt, security_mitigations, sos, stockpile, topology
 
---xz-single-threaded  
-Use single-threaded compression with 'xz'
+`--pbench-pre` STR  
+Path to the script which will be executed before tools are started  
+_NOTE: --pbench-pre is not compatible with --use-tool-triggers_
 
---show-server STR  
-Display information about the pbench server where the result(s) will be moved (Not implemented)
+`--pbench-post` STR  
+Path to the script which will be executed after tools are stopped and postprocessing is complete  
+_NOTE: --pbench-post is not compatible with --use-tool-triggers_
 
---help  
+`--use-tool-triggers`  
+Use tool triggers instead of normal start/stop around script;  
+_NOTE: --use-tool-triggers is not compatible with --iteration-list,--pbench-pre, or --pbench-post_
+
+`--no-stderr-capture`  
+Do not capture the stderr of the script to the result.txt file
+
+`--<command to run>`
+ It can be a script, or an executable, or a shell command 
+
+`--help`  
 Show this message and exit.
 
 ---
 
-### pbench-user-benchmark
+##### pbench-move-result
 
 ---
 
 **NAME**  
-pbench-user-benchmark, Control start/stop/post-process-tools.
+`pbench-move-results`, Move all results to 0.69 Pbench Server.
 
 **SYNOPSIS**  
-`pbench-user-benchmark[-C][--tool-group][--iteration-list][--sysinfo][--pbench-pre][--pbench-post][--use-tool-triggers][--no-stderr-capture]-- <script to run>`
+`pbench-move-results [--help] [--user=<user>] [--controller=<controller>] [--prefix=<path>] [--xz-single-threaded] [--show-server]`
 
 **DESCRIPTION**  
-Collects data from the tools registered. Here are the steps involved
-
-- Invoking pbench-user-benchmark with your workload generator as an argument: that will start the collection tools on all the hosts
-- Next, it will run your workload generator; when that finishes, it will stop the collection tools on all the hosts
-- Finally, the postprocessing phase will gather the data from all the remote hosts and run the postprocessing tools on everything.
+This command can only be used for a 0.69 Pbench Server destination, and requires a /opt/pbench-agent/id_rsa file with the private SSH key of the server's pbench account.
```

Comment:
This text, while important, is not a description of the command...so, I think we need to insert something before it (among other things, we should mention that this command _removes_ the result from the local system after uploading it...no?).

(And, `/opt/pbench-agent/id_rsa` should probably be in monospace.)